css/styles.css

```diff
@@ -1,189 +1,177 @@
-*, *::after,*::before{
-    box-sizing: border-box;
-}
-
-:root{
-    --primary-0:#FFFFFF;
-    --primary-100:#E7DEFE;
-    --primary-200:#CEBEFE;
-    --primary-500:#7C5CFC;
-    --secondary-300:#90A3BF;
-    --secondary-100:#E0E9F4;
-    --secondary-200:#C3D4E9;
-    --secondary-400:#596780;
-}
-
-
-html{  
-    font-size: 62.5%;
-    height: 100vh;
-    /* display: flex;
+*, *::after, *::before {
+  box-sizing: border-box; }
```

Comment:
The closing brace should be in new line. 

Are you following some standard for this?

But if you have shifted to scss, then we can skip making changes to this.